src/scenes/StageManager.ts

```diff
@@ -250,7 +256,7 @@ export default class StageManager{
 
     setScaling(scaleconfig:ScaleConfig) {
         if(scaleconfig.origin) {
-            this._originSize = this.getSize(scaleconfig.origin.width,scaleconfig.origin.height);
+            console.warn('origin is deprecated and will be ignored');
```

Comment:
I would argue that the code referenced here is obsolete (ie no longer working). https://stackoverflow.com/questions/9208091/the-difference-between-deprecated-depreciated-and-obsolete